internal/controller/changetransferpolicy_controller.go

```diff
@@ -342,6 +344,16 @@ func (r *ChangeTransferPolicyReconciler) creatOrUpdatePullRequest(ctx context.Co
 	}
 	prName := utils.KubeSafeUniqueName(ctx, utils.GetPullRequestName(ctx, gitRepo.Spec.Owner, gitRepo.Spec.Name, ctp.Spec.ProposedBranch, ctp.Spec.ActiveBranch))
 
+	promotionConfig := &promoterv1alpha1.PromotionConfiguration{}
+	if err := r.Get(ctx, client.ObjectKey{Name: r.Config.GlobalPromotionConfigurationName, Namespace: r.Namespace}, promotionConfig); err != nil {
+		return fmt.Errorf("failed to get global promotion configuration: %w", err)
+	}
+
+	title, body, err := TemplatePullRequest(&promotionConfig.Spec.PullRequest, ctp)
+	if err != nil {
+		return fmt.Errorf("failed to template pull request: %w", err)
+	}
```

Comment:
Can we isolate that in a new package even if it is the only usage for now. We kind of know that the logic to resolve the PromotionConfiguration object will get more complex in the future.
```
settingsMgr := settings.NewSettingsManager(globalPromotionConfigurationName, namespace)
promotionConfig, err := settingsMgr.GetPromotionConfiguration()
```

Reply:
Done!